internal/errors/worker_test.go

```diff
@@ -0,0 +1,311 @@
+//
+// Copyright (C) 2019-2021 vdaas.org vald team <vald@vdaas.org>
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+// Package errors provides error types and function
+package errors
+
+import (
+	"testing"
+
+	"go.uber.org/goleak"
+)
+
+func TestErrWorkerIsNotRunning(t *testing.T) {
+	t.Parallel()
+	type args struct {
+		name string
+	}
+	type want struct {
+		err error
+	}
+	type test struct {
+		name       string
+		args       args
+		want       want
+		checkFunc  func(want, error) error
+		beforeFunc func(args)
+		afterFunc  func(args)
+	}
+	defaultCheckFunc := func(w want, err error) error {
+		if !Is(err, w.err) {
+			return Errorf("got_error: \"%#v\",\n\t\t\t\twant: \"%#v\"", err, w.err)
+		}
+		return nil
+	}
+	tests := []test{
+		{
+			name: "return ErrWorkerIsNotRunning error when name is index",
```

Comment:
```suggestion
			name: "return an ErrWorkerIsNotRunning error when name is index",
```

Reply:
I think the current one is correct. What do you think?

Reply:
what's the current one?

Reply:
this one.
`return ErrWorkerIsNotRunning error when name is index`

Reply:
no article